components/ui/themes/DefaultTheme.ts

```diff
@@ -1573,7 +1573,7 @@ export const DefaultTheme = {
     // https://ethercreative.github.io/react-native-shadow-generator/
 
     if (level === 0) {
-      return null;
+      return {};
```

Comment:
what's the difference null vs empty object

Reply:
same as above